ozaria/site/components/teacher-dashboard/modals/ModalCurriculumPromotion.vue

```diff
@@ -0,0 +1,112 @@
+<script>
+import ModalDynamicPromotion from './ModalDynamicPromotion'
+import trackable from 'app/components/mixins/trackable.js'
+
+export default Vue.extend({
+  components: {
+    ModalDynamicPromotion
+  },
+  mixins: [trackable],
+  props: {
+    curriculumClicked: {
+      type: Boolean,
+      required: false
+    }
+  },
+  data: () => {
+    return {
+      showModal: true
+    }
+  },
+  computed: {
+    isOld () {
+      const twoDaysAgo = new Date(new Date() - 2 * 24 * 60 * 60 * 1000)
+      return new Date(me.get('dateCreated')) < twoDaysAgo
```

Comment:
let's also not show it dateCreated is after august 7. We don't need it for new teachers anyway.

Reply:
@gabriellui1 can you just implement this change for the PR? We will worry about other changes in a different feature.